spec/factories/apps.rb

```diff
@@ -0,0 +1,8 @@
+
+FactoryGirl.define do
+  factory :apns_app, class: 'RailsPushNotifications::APNSApp' do
+    apns_dev_cert   'abc'
+    apns_prod_cert  'def'
+    sandbox_mode    true
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.